.github/workflows/publish.yml

```diff
@@ -3,12 +3,28 @@ on:
   release:
     types: [published]
 
-env:
-  BRN_Version: 0.64
 
 jobs:
+  select-react-native-version:
+    runs-on: macos-latest
+    outputs:
+      version: ${{ steps.stepVersion.outputs.version }}
+    steps:
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: regex-match
+        with:
+          text: ${{ github.event.release.name }}
+          regex: '^(\d+\.)?(\d+)'
+      - name: Release infos
+        run: |
+          echo "- Tag ${GITHUB_REF/refs\/tags\//}"
+          echo "- Name: ${{ github.event.release.name }}"
+          echo "- Description: ${{ github.event.release.body }}"
+      - id: stepVersion
+        run: echo "::set-output name=version::${{ steps.regex-match.outputs.match }}"
```

Comment:
I don't follow what's happening on this line. When I saw the `echo` I thought it was just logging or something, but it looks very syntaxy so I'm guessing it's doing something else?

Reply:
Oh I think I see, this is some syntax for setting the `select-react-native-version` job output, which is accessed later via `needs.select-react-native-version.outputs.version`, correct?

Reply:
Yes, that's correct. The syntax for forwarding parameters is indeed special.